internal/controller/account_resolver.go

```diff
@@ -49,35 +43,17 @@ type BOPAccountIdResolver struct {
 
 func (bar *BOPAccountIdResolver) MapClientIdToAccountId(ctx context.Context, clientID domain.ClientID) (domain.Identity, domain.AccountID, error) {
 
-	fmt.Println("Looking up the connection's account number in BOP")
-	caCert, err := ioutil.ReadFile(bar.Config.BopCaFile)
+	logger.Log.Debugf("Looking up the client %s account number in via Gateway", clientID)
+	client := &http.Client{}
```

Comment:
I think by default the built-in http client does not have a timeout.  We do not have to hold up this PR up for that though.  I can add a timeout later.  I created https://issues.redhat.com/browse/RHCLOUD-12881 to make sure I don't forget to do this.

Reply:
Good point.  It was so easy to resolve I just pushed up an update..